.mergify.yml

```diff
@@ -36,9 +36,6 @@ pull_request_rules:
     conditions:
       - and:
           - status-success=all-web3-checks
-          - or:
-              - -files~=^packages/library-legacy/
-              - status-success="Validate commit message"
```

Comment:
This code lives on a separate branch now so we don't have to handle it.

Reply:
This isn't true. I actually do need this to work, but only if the branch is `maintenance/v1.x`.